pkg/attestation/filter.go

```diff
@@ -0,0 +1,51 @@
+package attestation
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/antonmedv/expr"
+)
+
+// FilteredEnv type including an array of Attestations
+type FilteredEnv struct {
+	Attestations []Attestation
+}
+
+// IsAttestationType indicates if a PURL has the given attestation type.
+func (a Attestation) IsAttestationType(t string) bool {
+	if attType, ok := a.PURL.Qualifiers.Map()["attestation_type"]; ok {
+		return strings.ToLower(attType) == t
+	}
+	return false
+}
+
+// RepositoryURLOf filters on a specific repository_url qualifier of the package.
+func (a Attestation) RepositoryURLOf(registryURL string) bool {
```

Comment:
Align the naming with the other filter method.

```suggestion
func (a Attestation) IsRegistry(registryURL string) bool {
```

Reply:
Good suggestion, thanks!